### DISCUSSION TITLE
Will Oceananigans change its name?

### DISCUSSION THREAD

**Discussion by iuryt** [CliMA/Oceananigans.jl#3043] [Ideas]
I can't help getting excited with the new project [DoubleTrouble](https://github.com/glwagner/DoubleTrouble.jl) and the amazing opportunities it brings me for coupled air-sea interactions using Oceananigans.
It made me think about if you have thought about changing the name of the model to something more general.
Fluidonanigans seem weird though. Luke SkyWater is cool but still doesn't cover the whole range of possibilities.
Is this something that you guys are planning?

Again, thanks for the awesome package!

**Comment by glwagner** [comment]
😄 we are hoping to keep Oceananigans mostly focused on ocean-flavored fluid dynamics! DoubleTrouble is just a toy for experimenting with user interfaces for coupled models --- but we certainly plan to couple Oceananigans to [ClimaAtmos](https://github.com/CliMA/ClimaAtmos.jl) eventually, so hopefully we will have a good platform for coupled simulations! We may use Oceananigans core tools to develop a sea ice model too --- but that would be in a separate repo, not in Oceananigans.jl.
